docs/source/tutorials/arch_modeling/spypads_tutorial.rst

```diff
@@ -0,0 +1,332 @@
+Creating Spypads Using XML Syntax
+=================================
+
+Introduction
+~~~~~~~~~~~~
+
+**In this tutorial, we will**
+   - Show the XML Syntax for Global Outputs

```

Comment:
It seems that some words are capitalized where they are not required. Please give a quick read to fix them. The rules are
- In the caption, only the first word should be capitalized (except terminologies)
- In sentences, only the first word should be capitalized (except terminologies)